src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -322,19 +315,14 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
       })
     );
 
-    div.setAttribute('data-title', this.output.title || '');
-
     if (this.vis.description) {
       div.setAttribute('data-description', this.vis.description);
     }
 
     div.setAttribute('data-test-subj', 'visualizationLoader');
     div.setAttribute('data-shared-item', '');
```

Comment:
i think this one should be added by render comlete as well (i think this indicates to reporting to wait for this embeddable to load (has implemented render complete stuff))

Reply:
Done.